README.md

```diff
@@ -245,3 +245,30 @@ render(html`<div>${until(request, timeout, loading)}</div>`)
 // ^ renders <div>Loading...</div>
 // After 2000ms will render <div>Failed to load</div>
 ```
+
+### CSP Trusted Types
+
+You can call `setCSPTrustedTypesPolicy(policy: TrustedTypePolicy | Promise<TrustedTypePolicy> | null)` from JavaScript to set a [CSP trusted types policy](https://web.dev/trusted-types/), which can perform (synchronous) filtering or rejection of the rendered template:
+
+```ts
+import {setCspTrustedTypePolicy} from "@github/jtml";
```

Comment:
```suggestion
import {setCSPTrustedTypesPolicy} from "@github/jtml";
```

Reply:
This is an old version of the code, the typo has been addressed in the most up to date version.